recipes_source/recipes/tuning_guide.py

```diff
@@ -236,6 +236,44 @@ def fused_gelu(x):
 
 # export LD_PRELOAD=<jemalloc.so/tcmalloc.so>:$LD_PRELOAD
 
+###############################################################################
+# Use oneDNN Graph with TorchScript for inference
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# oneDNN Graph can significantly boost inference performance. It fuses some compute-intensive operations such as convolution, matmul with their neighbor operations.
+# Currently, it's supported as an experimental feature for Float32 data-type.
+# oneDNN Graph receives the model’s graph and identifies candidates for operator-fusion with respect to the shape of the example input.
+# A model should be JIT-traced using an example input.
+# Speed-up would then be observed after a couple of warm-up iterations for inputs with the same shape as the example input.
+# The example code-snippets below are for resnet50, but they can very well be extended to use oneDNN Graph with custom models as well.
+
+# Only this extra line of code is required to use oneDNN Graph
+torch.jit.enable_onednn_fusion(True)
+
+###############################################################################
+# Using the oneDNN Graph API requires just one extra line of code.
+# As an aside, ``torch.jit.optimize_for_inference`` should not be called if oneDNN Graph is to be used.
```

Comment:
@sanchitintel can you please explain what do you mean here by "As an aside"?

Reply:
Can be removed completely. Example rephrase: 

If you are using oneDNN Graph, avoid calling `torch.jit.optimize_for_inference``.

Reply:
Thanks for the review, @malfet & @svekars!
I've revised the statement. I just meant it's a digression from the line above it.